docs/usage/pivoting.md

```diff
@@ -13,17 +13,28 @@ In your workspace, you can add entities that you want to investigate, visualize
 
 ![Investigation workspace](assets/investigation-workspace.png)
 
+On some nodes, you can see next to them a bullet with a number inside. It is a visual indication that shows you how many entities have relationship with this one.
```

Comment:
Why some ? Is not for everyone ?

Reply:
There is no bullet displayed when nothin to expand, but maybe I can just remove "On some nodes" if it is confusing

Reply:
Removed